core/src/main/java/org/elasticsearch/index/query/MissingQueryBuilder.java

```diff
@@ -51,6 +69,10 @@ public MissingQueryBuilder nullValue(boolean nullValue) {
         return this;
     }
 
+    public boolean nullValue() {
```

Comment:
Reading just the method name, I was curious what it means in this context. Maybe short comment?